simple/templates/rollout.yaml

```diff
@@ -5,16 +5,8 @@ metadata:
   name: {{ .Values.name }}
   annotations:
 {{- toYaml .Values.rollout.annotations | nindent 4 }}
-spec: 
-  {{- if .Values.hpa }}
-  replicas: {{ .Values.hpa.minReplicas }}
-  {{- else if .Values.hpav2 }}
-  replicas: {{ .Values.hpav2.minReplicas }}
-  {{- else if .Values.KEDA}}
-  replicas: {{ .Values.KEDA.ScaledObject.spec.minReplicaCount | default .Values.replicaCount }}
-  {{- else }}
-  replicas: {{ .Values.replicaCount }}
-  {{- end }}
+spec:
+  replicas: {{ .Values.rollout.replicas }}
```

Comment:
as my knowledge, we need to prevent any replica count set in this value which may cause replica dropped to the minimum value when Rollout custom resource first created or modified.
I think it will be better if this value is required and do notice developers what kind of value they should fill in.